components/hackathon/common/AvatarShowcase.tsx

```diff
@@ -20,13 +20,12 @@ export const AvatarShowcase: React.FC<AvatarShowcaseProps> = ({
   return (
     <VStack mb={20} spacing={12}>
       <Heading size="title.2xl">{title}</Heading>
-      <SimpleGrid
```

Comment:
why? these makes the columns not the same size
![image](https://user-images.githubusercontent.com/50103937/211381602-2475f36c-aea2-4e53-98cd-553869073729.png)

Reply:
we won't be able to center it otherwise, right?

Reply:
is that super relevant? I very much prefer it to be aligned than centered